app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -23,28 +23,32 @@
       <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/profile_chooser_container"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingTop="20dp">
+        android:layout_height="match_parent">
 
         <TextView
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:layout_marginEnd="4dp"
-          android:minHeight="24dp"
+          android:fontFamily="sans-serif"
+          android:gravity="center"
+          android:minHeight="48dp"
           android:text="@string/profile_chooser_language"
           android:textColor="@color/profileChooserGreyTextColor"
-          android:fontFamily="sans-serif"
           android:textSize="16sp"
+          android:visibility="gone"
           app:layout_constraintEnd_toStartOf="@+id/profile_chooser_language_icon"
           app:layout_constraintTop_toTopOf="parent" />
 
         <ImageView
           android:id="@+id/profile_chooser_language_icon"
-          android:layout_width="24dp"
-          android:layout_height="24dp"
+          android:layout_width="48dp"
+          android:layout_height="48dp"
           android:layout_marginEnd="36dp"
-          android:fontFamily="sans-serif"
+          android:paddingLeft="4dp"
+          android:paddingTop="12dp"
+          android:paddingRight="20dp"
```

Comment:
Instead of paddingRight use paddingStart and End.

Reply:
Done